visallo/visallo
# getOntology Should Return PUBLIC for a null Workspace ID

Since the [workspace ID is the cache key](https://github.com/visallo/visallo/pull/1319), when null is provided to `OntologyRepositoryBase.getOntology` as the workspace ID an error is thrown. In order to properly handle this, the PUBLIC ontology will be returned if no workspace ID is provided.


Testing Instructions:
1. Log into Visallo as a new user
2. Ensure that the login succeeds and there are no error messages in the logs
